test/jackdaw/client_test.clj

```diff
@@ -104,7 +104,7 @@
 
 
 (deftest ^:integration send!-test
-  (fix/with-fixtures [(fix/topic-fixture (broker-config) test-topics 1000)]
+  (fix/with-fixtures [(fix/topic-fixture (broker-config) test-topics {:timeout-ms 1000})]
```

Comment:
This is the new way to express a timeout if you want something different from the default. The old way still works but should be considered deprecated.